contracts/examples/PayExample.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.20;
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract PayExample is Ownable {
+    /**
+     * @dev mapping of paymentValueId - keccak256(abi.encode(issuerId, schemaHash)) => value
+     */
+    mapping (bytes32 paymentValueId => uint256 value) private valueToPay;
+
+     /**
+     * @dev mapping of paymentRequest - keccak256(abi.encode(issuerId, paymentId)) => bool
+     */
+    mapping (bytes32 paymentRequest => bool isPaid) public payments;
```

Comment:
May be `paymentRequestId` is better than `paymentRequest`

Reply:
renamed